### PR TITLE
Updated twitter README.md

### PR DESCRIPTION
**Pull request by desojo**
Added a missing ` and updated title

### CODE REVIEW

File: src/autogpt_plugins/twitter/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# autogpt-twitter 🐣
+# desojo/autogpt-twitter 🐣
```
Comment (ntindle) [inline src/autogpt_plugins/twitter/README.md:1]: I'd add your name as a maintainer somewhere lower so people don't ping you every 3 seconds asking for help :)

Comment (ntindle) [reply src/autogpt_plugins/twitter/README.md:1]: Don't ask how I know

Comment (desojo) [reply src/autogpt_plugins/twitter/README.md:1]: Oh I'm fine with the visibility lol, we could also add maintainers as a column next to the plugin name within the base README.md if you'd rather

Comment (desojo) [reply src/autogpt_plugins/twitter/README.md:1]: With how the last PR was merged due to me being a git noob, I'd like to at least be listed as a contributor through this readme update if anything 😄

Comment (ntindle) [reply src/autogpt_plugins/twitter/README.md:1]: yeah add it to the root  readme table

**Pull request merged**